tests/test_core.py

```diff
@@ -250,17 +250,26 @@ def test_accessors(self):
         self.assertEqual(d['A'], 2.0)
         self.assertEqual(d['B'], 1.5)
 
+        #test using `update()`
```

Comment:
Can you make a separate test for the `update` method? It would also be good to check: updating for only a subset of the keys (rather than both), and to check that the appropriate behavior happens if both *args and **kwargs are used.